test/clj/rems/test_browser.clj

```diff
@@ -20,91 +19,22 @@
             [rems.browser-test-util :as btu]
             [rems.config]
             [rems.db.test-data-helpers :as test-helpers]
-            [rems.db.test-data-users :as test-users]
-            [rems.db.user-settings :as user-settings]
             [rems.db.users :as users]
+            [rems.db.user-settings :as user-settings]
             [rems.standalone]
             [rems.testing-util :refer [with-user]]
-            [rems.text :as text]
-            [rems.db.api-key :as api-key]
-            [rems.db.test-data :as test-data]))
+            [rems.text :as text]))
 
 (comment ; convenience for development testing
   (btu/init-driver! :chrome "http://localhost:3000/" :development))
```

Comment:
does one need to run (create-test-data) once on their non-test db for the interactive mode to work?

Reply:
Since it is run against normal db and backend at port 3000 you should have data there already. At least until example data becomes different.

Reply:
yeah but the data might be different... maybe it would be best to switch to a model where the tests create all data they need. for example the workflow licenses and draft application probably shouldn't be in create-test-data

Reply:
If there is more, or you have manually removed something. I can live with that. It is supposed to be the same right now. 

The decided future direction is to make all the tests independent of common data. And most of the browser tests already create the test data (sans users really). I think we can live with this situation but definitely the way forward is to make them completely independent.

Reply:
The original problem that this fixes was that the test data was created too many times and this solves it by not creating it because it should exist in your dev instance. Dev instance is used so that you know what is in the database and can easily verify it using the UI or some SQL tools. And can iterate the test statements one by one.